cogs/utils/errors.py

```diff
@@ -0,0 +1,42 @@
+from discord.ext import commands
+
+
+class PostException(Exception):
+    """
+    Raised when there's a error in the post()
+    wrapper
+    """
```

Comment:
Single line docstrings should only take a single line, also, fix a grammatic mistake (`a` should be `an`).
```suggestion
    """Raised when there's an error in the post() wrapper."""
```